src/client/Ruby/ms-rest-azure/lib/ms_rest_azure/resource.rb

```diff
@@ -23,6 +23,10 @@ class Resource
     # @return [Hash{String => String}] the tags attached to resources (optional).
     attr_accessor :tags
 
+    # @return [String] the kind of the resource.
+    # FIXME kind attribute was introduced because of SiteInstance model of azure_mgmt_web gem has kind attribute.

```

Comment:
Kind is in this [storage spec](https://github.com/Azure/azure-rest-api-specs/blob/master/arm-storage/2016-01-01/swagger/storage.json). Thoughts?